examples/crowdfunding/test/crowdfundingTest.js

```diff
@@ -0,0 +1,252 @@
+import {
+  addressToBytes,
+  getProgram,
+  intToBigEndian,
+  SignType,
+  stringToBytes,
+  TransactionType
+} from '@algorand-builder/algob';
+import { Runtime, StoreAccountImpl } from '@algorand-builder/algorand-js/build/src/index';
+import { assert } from 'chai';
+
+import { getAcc } from './common';
```

Comment:
can we reuse this function from `algorand-js`?

Reply:
again it is in `test`. will have to move and update

Reply:
Aren't we using this function in tests only?

Reply:
it is present in `algorand-js/test` if we want to use it here. then we have to first move it to `algorand-js/src` (after that put it in index.js), then only we will be able to properly import it here.

Reply:
Alright. Also, in line 9 why are we importing function from `build/`?

Reply:
@ratik21 that's a bug, will be done in other task. couldn't figure a solution.

Reply:
Okay